sphinx_view/__init__.py

```diff
@@ -7,4 +8,13 @@
 __all__ = [
     "__version__",
     "DocumentationView",
+    "JSONTOCHTMLBuilder",
 ]
+
+
+def setup(app):
+    app.add_builder(JSONTOCHTMLBuilder)
```

Comment:
I think we can pass `override=True` to use `name = "json"`, and then `make json`. 

🤔

Reply:
I believe you are correct [[source](https://www.sphinx-doc.org/en/master/_modules/sphinx/application.html#Sphinx.add_builder)]

Reply:
Confirming the the [new code](https://github.com/carltongibson/django-sphinx-view/pull/15/commits/6d3c1a6bfb22e1f56ecf0bb480c9d529963867a7) worked for me on a test project.

For the double rendering of the TOC, it seems the new "toctree" variable puts together an entire toctree, but if there is a toctree directive inside a document, that will render in the document as well.

The only thing I've seen to still be clean is a case where none of my leaves in the content tree have their own leaves. Said another way, there is one single toctree directive and it's in the ./index.rst file. The root level index page will still render [1] the new toctree variable in my sidebar as well as [2] the ./index.rst toctree directive inside the main container.

Reply:
> For the double rendering of the TOC... 

Let me have a think about that. 🤔